Makefile

```diff
@@ -1,6 +1,6 @@
 
-# Image URL to use all building/pushing image targets
-IMG ?= kubevirt-ipam-controller:latest
+# Default to registry path for consistency with CI/CD
+IMG ?= ghcr.io/kubevirt/ipam-controller:$(shell ./hack/sanitize-branch.sh $(shell git rev-parse --abbrev-ref HEAD))
```

Comment:
not sure I like this ...

Let me see if I'm understanding this properly .. 
For me to know what name the img will take (in the pre-submit stage), I'll need to check the contents of the install manifest, and see what name it defines in it ?